src/duiker.py

```diff
@@ -380,7 +380,7 @@ def parse_args(argv):
         PS1="\\u@\\h:\\w$ "
     }}
 
-    PROMPT_COMMAND=__prompt
+    PROMPT_COMMAND='(__prompt $)'
```

Comment:
That's an interesting idea. I agree that we should shorten `duiker import` execution time (or the perception of it) as much as possible.

I've never seen the `$` notation and can't find any reference in `bash(1)`, the [Advanced Bash Scripting Guide](http://www.tldp.org/LDP/abs/html/abs-guide.html), or the [BashGuide](http://mywiki.wooledge.org/BashGuide/). Mind linking to some documentation? I'm pretty curious about the dark corners of Bash.